client/App.tsx

```diff
@@ -1,45 +1,33 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { useState } from 'react';
 import CategoriesScreen from './src/components/CategoriesScreen';
 import HomeScreen from './src/components/HomeScreen';
 import IntroScreen from './src/components/IntroScreen';
-import { emptyUser } from './src/constants/emptyMocks';
-import { User } from './src/constants/interfaces';
 
 export type StackParamList = {
-  Home: {
-    user: User;
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Categories: {
-    user: User;
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Intro: {
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
+  Home: undefined;
```

Comment:
I believe we could omit this type, cause it doesn't help

Reply:
Specifying undefined means that the route doesn't have params. This is from docs: https://reactnavigation.org/docs/typescript/